### DISCUSSION TITLE
Fixed Link to Page over uid

### DISCUSSION THREAD

**Discussion by sandrosuter** [dotnet/docfx#7987] [General]
Is it possible to pass docfx generated page a uid or something else so that then automatically is called to the corresponding page?
The goal is to have a fixed link from external, even if the structure in the documentation changes.

I first thought xref was the solution, but if I understand this correctly it is only to link 2 manuals.

**Comment by KalleOlaviNiemitalo** [comment]
DocFX v3 has some support for `.openpublishing.redirection.json` files ([example in azure-docs](https://github.com/MicrosoftDocs/azure-docs/blob/0045f780bc732c4888ade6dfc8f43bf6e75c0bda/.openpublishing.redirection.json)) and `redirections.yml` files, which might be suitable for this, but I don't know whether `uid` can be referenced in those.

With DocFX v2, I guess you could let DocFX generate xrefmap.yml, and then make a custom tool transform that to something supported by your HTTP server software.
